### PR TITLE
Safari hat In-hand Sprites

### PR DESCRIPTION
**Pull request by onesch**
## About the PR
Adds in-hand sprites to the safari hat. based [#33386](https://github.com/space-wizards/space-station-14/issues/33386)

## Why / Balance
Currently it has no in-hands, so this is adding one.

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

### CODE REVIEW

File: Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+      {
+        "name": "inhand-left",
+        "directions": 4
+      },
+      {
+        "name": "inhand-right",
+        "directions": 4
+      }
+    ]
+  }
```
Comment (slarticodefast) [inline Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json:26]: Please keep the indentation as it was before at two spaces per level.
Also make sure the closing curly brackets line up correctly.
I'll ask an art director for approval for the sprite.

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Comment by slarticodefast** [comment]
Also a small visual change like this does not need a changelog entry, could you remove that from your PR description?
You will ofc course still be mentioned in the game's credits for contributing.

**Review by Emisse — APPROVED** [review]

**Comment by onesch** [comment]
> Also a small visual change like this does not need a changelog entry, could you remove that from your PR description?
> You will of course still be mentioned in the game's credits for contributing.

I deleted the changelog.

Is the problem with indents solved? Is the fact that I approved your changes regarding indents already in the branch as a commited?

**Comment by slarticodefast** [comment]
It seems like you did not push the changes to your branch correctly, the indentation hasn't been fixed yet.

**Review by onesch — COMMENTED** [review]
Correct?

**Review by slarticodefast — APPROVED** [review]
Looks good to me, thank you for your contribution!

**Pull request merged**